docs/zopebook/Sessions.rst

```diff
@@ -1,6 +1,27 @@
 Session Management
 ##################
 
+Sessions in Zope have historically been server side. They are simple to use, 
+and (usually) only rely on one cookie that contains the browser id. However, 
+server side sessions are hard to scale in horizontal deployments where you are 
+using ZEO, or have many Zope servers with read only ZODBs. Also the historical 
+backend implementation of sessions 
+`tempstorage <https://github.com/zopefoundation/tempstorage>`_ is broken and 
+deprecated so server side sessions cannot be used 'as is' anymore in Zope 4+.
+
+For this reason, and especially if you need only very few small session values, 
+consider using cookies as a replacement. The API for that would be 
+``REQUEST.get()`` and ``RESPONSE.setCookie()``. Plone has 
+`a bit more documentation on how to work with cookies <https://docs.plone.org/develop/plone/sessions/cookies.html>`_.
```

Comment:
Now I know where you got that `REQUEST.cookies` thing from for looking up cookie values. I would still not consider that part of the official API, but there's no point in showing different information here and in the Plone document, so I would change `REQUEST.get()` above to the same `REQUEST.cookies.get()" used in the Plone document.

Reply:
[x] done (push to come)